includes/multi-currency/Compatibility/WooCommerceNameYourPrice.php

```diff
@@ -140,4 +137,43 @@ public function should_convert_product_price( bool $return, $product ): bool {
 
 		return $return;
 	}
+
+	/**
+	 * Add currency to cart edit link.
+	 *
+	 * @param array $args      The cart args.
+	 * @param array $cart_item The current cart item.
+	 *
+	 * @return array
+	 */
+	public function edit_in_cart_args( $args, $cart_item ) {
+		$args['nyp_currency'] = $this->multi_currency->get_selected_currency()->get_code();
+		return $args;
+	}
+
+	/**
+	 * Maybe convert any prices being edited from the cart
+	 *
+	 * @param string $initial_price The initial price.
+	 * @param mixed  $product       The product being queried.
+	 * @param string $suffix        The suffix needed for composites and bundles.
+	 *
+	 * @return float|string
+	 */
+	public function get_initial_price( $initial_price, $product, $suffix ) {
+
+		if ( isset( $_REQUEST[ 'nyp_raw' . $suffix ] ) && isset( $_REQUEST['nyp_currency'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
I was thinking if we can combine the isset checks and it turns we can do this actually. So, instead of 

```
if ( isset( $_REQUEST[ 'nyp_raw' . $suffix ] ) && isset( $_REQUEST['nyp_currency'] ) )
```

We could write 

```
isset($_REQUEST['nyp_raw' . $suffix], $_REQUEST['nyp_currency'])
```

And this works the same. The `isset` still returns true if both `$_REQUEST['nyp_raw' . $suffix]` and `$_REQUEST['nyp_currency']` are set and not null. 

Do you think we could implement this?

Reply:
Keeping the separate `isset` requests makes the code easier to read when skimming through it. Either would work, though.